cypress/support/commands.ts

```diff
@@ -0,0 +1,32 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+//
+// For more info about typing custom commands,
+// read more here:
+// https://docs.cypress.io/guides/tooling/typescript-support#Types-for-custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+/// <reference types="@testing-library/cypress" />
```

Comment:
This shouldn't be necessary by including `"@testing-library/cypress"` in `tsconfig.json` within cypress folder, but doesn't seem to work correctly. Adding this as a workaround meanwhile, I'll have to open an issue in that package.

Reply:
Can you leave a comment in the code explaining this? That way I can see that later when I'm wondering why we added that. 😉